extensions/multisuite/suite.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multisuite
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/networkservicemesh/gotestmd/pkg/suites/shell"
+)
+
+type Suite struct {
+	shell.Suite
+
+	suits []suite.TestingSuite
+}
+
+func (s *Suite) WithSuits(suits ...suite.TestingSuite) {
```

Comment:
Please consider making this a constructor to prevent problems with non expected modifying `suits`.